support-services/src/test/scala/com/gu/support/redemption/corporate/CorporateCodeValidatorSpec.scala

```diff
@@ -13,91 +13,91 @@ class CorporateCodeValidatorSpec extends AsyncFlatSpec with Matchers {
     val codeValidator = CorporateCodeValidator.withDynamoLookup {
-      case "CODE" => Future.successful(Some(Map(
+      case "test-code-123" => Future.successful(Some(Map(
         "available" -> DynamoBoolean(true),
         "corporateId" -> DynamoString("1")
       )))
     }
-    codeValidator.getStatus(RedemptionCode("CODE").right.get).map {
+    codeValidator.getStatus(RedemptionCode("test-code-123").right.get).map {
       _ should be(ValidCorporateCode(CorporateId("1")))
     }
   }
 
   it should "handle an NON available code" in {
     val codeValidator = CorporateCodeValidator.withDynamoLookup {
-      case "CODE" => Future.successful(Some(Map(
+      case "test-code-123" => Future.successful(Some(Map(
         "available" -> DynamoBoolean(false),
         "corporateId" -> DynamoString("1")
       )))
     }
-    codeValidator.getStatus(RedemptionCode("CODE").right.get).map {
+    codeValidator.getStatus(RedemptionCode("test-code-123").right.get).map {
       _ should be(CodeAlreadyUsed)
     }
   }
 
   it should "handle an NON EXISTENT code" in {
     val codeValidator = CorporateCodeValidator.withDynamoLookup {
-      case "CODE" => Future.successful(None)
+      case "test-code-123" => Future.successful(None)
     }
-    codeValidator.getStatus(RedemptionCode("CODE").right.get).map {
+    codeValidator.getStatus(RedemptionCode("test-code-123").right.get).map {
       _ should be(CodeNotFound)
     }
   }
 
   it should "handle an code with invalid available type " in {
     val codeValidator = CorporateCodeValidator.withDynamoLookup {
-      case "CODE" => Future.successful(Some(Map(
+      case "test-code-123" => Future.successful(Some(Map(
         "available" -> DynamoString("haha not a boolean"),
         "corporateId" -> DynamoString("1")
       )))
     }
     recoverToSucceededIf[RuntimeException] {
-      codeValidator.getStatus(RedemptionCode("CODE").right.get)
+      codeValidator.getStatus(RedemptionCode("test-code-123").right.get)
     }
   }
 
   it should "handle an code with invalid corporate id type " in {
     val codeValidator = CorporateCodeValidator.withDynamoLookup {
-      case "CODE" => Future.successful(Some(Map(
+      case "test-code-123" => Future.successful(Some(Map(
         "available" -> DynamoBoolean(true),
         "corporateId" -> DynamoBoolean(false)
       )))
     }
     recoverToSucceededIf[RuntimeException] {
-      codeValidator.getStatus(RedemptionCode("CODE").right.get)
+      codeValidator.getStatus(RedemptionCode("test-code-123").right.get)
     }
   }
 
   it should "handle a missing attribute code - available" in {
     val codeValidator = CorporateCodeValidator.withDynamoLookup {
-      case "CODE" => Future.successful(Some(Map(
-        "ASDFNQWEOIDNSDKNFNAKNDAKNSKANSDKNASDAKSND" -> DynamoBoolean(true),
+      case "test-code-123" => Future.successful(Some(Map(
+        "asdfnqweoidnsdknfnakndaknskansdknasdaksnd" -> DynamoBoolean(true),
```

Comment:
this is not a code, it's just a random attribute name in place of `"available"`

Reply:
consider calling it "arbitraryStrangeAttributeName" or something

Reply:
Ah well, might as well go lower case!